tests/test_integration/helpers.py

```diff
@@ -0,0 +1,62 @@
+from time import time
+from uuid import uuid4
+
+
+def create_partner(service, **custom_attrs):
+        # can't use a uuid because of 30 char limit:
+        unique_id = str(time()).replace('.', '')
+
+        attrs = {
+            'id': 'WL_TEST-%s' % unique_id,
+            'name': 'TEST',
+            'parent_partner_id': 'WL_YOLA',
```

Comment:
Good catch. Fixed hard-coded partner ids in https://github.com/yola/yolapy/commit/86380d687b3f8b6e9dfad78ef6fb9523953474eb